l1-contracts/test/portals/TokenPortal.sol

```diff
@@ -93,6 +133,50 @@ contract TokenPortal {
     underlying.transfer(msg.sender, _amount);
     return entryKey;
   }
+
+  /**
+   * @notice Cancel a private depositToAztec L1 to L2 message
+   * @dev only callable by the `canceller` of the message
+   * @param _amount - The amount to deposit per the original message
+   * @param _deadline - The timestamp after which the entry can be cancelled
+   * @param _secretHashForL2MessageConsumption - The hash of the secret consumable L1 to L2 message
+   * @param _secretHashForL2MessageConsumption - The hash of the secret to redeem minted notes privately on Aztec
+   * @param _fee - The fee paid to the sequencer
+   * @return The key of the entry in the Inbox
+   */
+  function cancelL1ToAztecMessagePrivate(
+    uint256 _amount,
+    uint32 _deadline,
+    bytes32 _secretHashForL2MessageConsumption,
+    bytes32 _secretHashForRedeemingMintedNotes,
+    uint64 _fee
+  ) external returns (bytes32) {
+    // @todo: (issue #624) handle different versions
```

Comment:
You can get rid of these, think they are stale, or at least not relevant while we don't know the exact upgrade mechanism if any.

Reply:
Similar for the withdraw.